### PR TITLE
Added support for Puppet

### PR DESCRIPTION
**Pull request by 6uhrmittag**
I added Puppet to the scope.

Honestly, I have no idea if this is enough - but I was able to build the Plugin locally and use it.

To build it locally, I needed to Change the `kotlin_version `as well. 
https://github.com/kevinlinxp/intellij-sublime-snippets-support/blob/master/build.gradle#L2
````
    ext.kotlin_version = '1.3.72'
````

In any case, thank you for this plugin! It looks like it's the best way to manage live templates.

### CODE REVIEW

File: src/main/kotlin/com/kevinlinxp/sublimeSnippetsSupport/SublimeSnippetScope.kt [hunk 1]
```diff
@@ -30,6 +30,7 @@ enum class SublimeSnippetScope(private val scopeName: String, private val contex
     SOURCE_TSX("source.tsx", "TypeScript"),
     SOURCE_LUA("source.lua", "LUA"),
     SOURCE_PUPPET("source.puppet", "PUPPET_FILE"),
+    SOURCE_OTHER("source.other", "OTHER"),
```
Comment (kevinlinxp) [inline src/main/kotlin/com/kevinlinxp/sublimeSnippetsSupport/SublimeSnippetScope.kt]: The scope of "OTHER" is ambiguous per IDE and per individual. So it's better not to add this scope which may bring in unwanted snippets.

**Comment by kevinlinxp** [comment]
Hi @6uhrmittag,

Thank you very much for the pull request.

I am glad you find the plugin useful :)

Please have a look at my comment for the "OTHER" scope.

Cheers,
Kevin

File: src/main/kotlin/com/kevinlinxp/sublimeSnippetsSupport/SublimeSnippetScope.kt (hunk 1 above)
Comment (6uhrmittag) [reply src/main/kotlin/com/kevinlinxp/sublimeSnippetsSupport/SublimeSnippetScope.kt]: You're right  right! I removed it from this request

**Comment by 6uhrmittag** [comment]
Woops, Sorry! I removed the 2 out-of-scope changes. This Request is supposed to only add the line for puppet.

**Review by kevinlinxp — APPROVED** [review]
Many thanks @6uhrmittag.

I merged the change and will do another release within this week :)

**Pull request merged**